packages/components/src/components/hds/pagination/numbered/index.hbs

```diff
@@ -33,7 +32,7 @@
               <Hds::Pagination::Nav::Ellipsis />
             {{else}}
               <Hds::Pagination::Nav::Number
-                @page={{page}}
+                @page={{this.elliptizedPageArrayItemAsNumber page}}
```

Comment:
Out of curiosity, why did this change?

Reply:
Glint isn't smart enough to figure out the conditional above this line where we check if the value is an ellipsis, and only render the `Hds::Pagination::Nav::Number` component if it's not. The `Number` component always expects `@page` to be a number, so we need to run the value through a function that returns the value as a number. It's a little hacky, but there's no other way to cast a type in Glimmer that I'm aware of.